src/components/Campaign.tsx

```diff
@@ -70,19 +95,36 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
+                <li>
+                  <button
+                    type="button"
+                    onClick={showModal}
+                    className="dp-button button-medium primary-green"
+                  >
+                    <FormattedMessage id="save_template" />
+                  </button>
```

Comment:
I think that this button should not be shown when the content is HTML, we do not support it, right?

Reply:
Is ok for me, I will dissapear the button when content is html.